.travis.yml

```
@@ -28,17 +28,19 @@ jobs:
     - julia: 1.3
       env:
         - BINARYBUILDER_RUNNER=privileged
-        - BINARYBUILDER_USER_SQUASHFS=true
+        - BINARYBUILDER_USE_SQUASHFS=true
 
     # Add a job that uses the unprivileged builder with unpacked shards
-    - julia: 1.3
+    - julia: 1.3.1
```

I meant
```suggestion
    - julia: 1.3.0
```
:man_facepalming: